frontend/src/components/browse-sightings-page/BrowseSightings.tsx

```diff
@@ -1,24 +1,38 @@
 import React, { useEffect, useState } from "react";
-import { getSightings, Sighting } from "../../clients/apiClient";
+import {
+  getExternalSightings,
+  getSightings,
+  Sighting,
+  SightingFromExternalApi,
```

Comment:
This has been renamed now:
```suggestion
  ExternalSighting,
```